packages/opentelemetry-core/test/common/ConsoleLogger.test.ts

```diff
@@ -0,0 +1,120 @@
+/**
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as assert from 'assert';
+import * as logger from '../../src/common/ConsoleLogger';
+import { LogLevel } from '../../src/common/types';
+
+describe('ConsoleLogger', () => {
+  const origDebug = console.debug;
+  const origInfo = console.info;
+  const origWarn = console.warn;
+  const origError = console.error;
+  let debugCalled = false;
+  let infoCalled = false;
+  let warnCalled = false;
+  let errorCalled = false;
+
+  beforeEach(() => {
+    // mock
+    console.debug = () => {
+      debugCalled = true;
```

Comment:
Should we be checking the arguments that debug was called with, e.g.

```
console.debug = (...args) => {
   debugCalledArgs = args;
}
```
Then you can verify that it gets the right parameters

Reply:
Sure, done. ptal